spec/chewy/query_spec.rb

```diff
@@ -454,7 +456,7 @@
   end
 
   describe '#exists?' do
-    let(:data) { 10.times.map { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
+    let(:data) { Array.new(10) { |i| {id: i.next.to_s, name: "Name#{i.next}", age: 10 * i.next}.stringify_keys! } }
```

Comment:
Oh, really? Don't think it is significant in this particular case, `10.times.map` looks more like ruby-way as for me, `Array.new(10)` is c++ or java.